src/spring-cloud/setup.py

```diff
@@ -56,5 +56,5 @@
     classifiers=CLASSIFIERS,
     packages=find_packages(),
     install_requires=DEPENDENCIES,
-    package_data={'azext_spring-cloud': ['azext_metadata.json']},
+    package_data={'azext_spring_cloud': ['azext_metadata.json']},

```

Comment:
https://github.com/Azure/azure-cli-extensions/blob/1941e08c319f2ee5bb69f8d3f266295f7afba2d5/src/ip-group/setup.py#L56

Looks like this ext is also using `-`. Which one is correct?